Sources/Overload/OvEditor/src/OvEditor/Core/Context.cpp

```diff
@@ -8,13 +8,59 @@
 
 #include <OvRendering/Entities/Light.h>
 #include <OvCore/Global/ServiceLocator.h>
+#include <OvDebug/Assertion.h>
 #include <OvTools/Utils/SystemCalls.h>
 
 #include "OvEditor/Core/Context.h"
 
 using namespace OvCore::Global;
 using namespace OvCore::ResourceManagement;
 
+constexpr std::array<std::pair<int, int>, 13> kResolutions
+{
+	std::make_pair(640, 360), // nHD
+	std::make_pair(854, 480), // FWVGA
+	std::make_pair(960, 540), // qHD
+	std::make_pair(1024, 576), // WSVGA
+	std::make_pair(1280, 720), // HD
+	std::make_pair(1366, 768), // FWXGA
+	std::make_pair(1600, 900), // HD+
+	std::make_pair(1920, 1080), // Full HD
+	std::make_pair(2560, 1440), // QHD
+	std::make_pair(3200, 1800), // QHD+
+	std::make_pair(3840, 2160), // 4K UHD
+	std::make_pair(5120, 2880), // 5K
+	std::make_pair(7680, 4320), // 8K UHD
+};
+
+std::array<int, 4> FindBestFitWindowSizeAndPosition(std::array<int, 4> p_workAreaSize)
```

Comment:
Can be static

Reply:
The only benefit of making it `static` would be to restrict this symbol to the `Context.cpp` translation unit. We haven't historically used that approach in Overload, but it could be a good way to avoid function name collisions.

The use of the `static` keyword to restrict a function to a single translation unit has been debated and even considered for deprecation; however, it is still supported as of today. I'm not sure if it's a future-proof solution.

Another way to achieve the same result (restricting the symbol to the current translation unit) is to declare the function in an anonymous namespace:
```cpp
namespace
{
    void foo() {}
}
```